Source/WebCore/page/ResizeObservation.cpp

```diff
@@ -41,6 +41,7 @@ Ref<ResizeObservation> ResizeObservation::create(Element& target, ResizeObserver
 
 ResizeObservation::ResizeObservation(Element& element, ResizeObserverBoxOptions observedBox)
     : m_target { element }
+    , m_lastObservationSizes { LayoutSize(-1, -1), LayoutSize(-1, -1), LayoutSize(-1, -1) }
     , m_observedBox { observedBox }
```

Comment:
This seems to work for `check-webkit-style`:
```suggestion
    , m_observedBox { observedBox }
    , m_lastObservationSizes { LayoutSize(-1, -1), LayoutSize(-1, -1), LayoutSize(-1, -1) }
```

Reply:
I don't think style checker failing matters much.

Reply:
Mm, yeah, I just didn't want it to start permafailing, but it seems it only checks problems in the current commit, so I guess it will be fine.

Reply:
We need to follow the order of declaration in the head file, otherwise the compiler complains.
So I'd prefer to leave as it was.